bin/copy-production-modules.js

```diff
@@ -96,6 +97,9 @@ function processPackage( pkgName, context ) {
 		);
 
 		if ( foundFolderIdx === -1 ) {
+			if ( optionalPkgDeps.includes( depName ) ) {
+				continue;
+			}
```

Comment:
This was needed to allow us to skip optional dependencies, defined through [peerDependenciesMeta field](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta).

`jsdom` has an optional dependency canvas that has trouble building, resulting in the error:
```
Error: dlopen(/.../node_modules/canvas/build/Release/canvas.node, 0x0001): symbol not found in flat namespace (_cairo_fill)
```
This is most likely causing issues with TeamCity builds as well.

Fortunately, this [is optional](https://github.com/jsdom/jsdom/blob/master/package.json#L57) and skipping it shouldn't affect the sanitizing process.

Reply:
That makes sense to me. Here's some more info about Canvas in jsdom: https://github.com/WordPress/gutenberg/issues/23947. It looks like we would have to include the dependency ourselves for it to work.

Reply:
plus, if we aren't allowing the `canvas` tag in the sanitizer anyways, we wouldn't have any use for the dependency